src/ess/livedata/handlers/monitor_data_handler.py

```diff
@@ -3,41 +3,52 @@
 from __future__ import annotations
 
 from collections.abc import Hashable
-from typing import NewType
 
 import numpy as np
 import scipp as sc
 
-from ess.reduce.nexus.types import Filename, NeXusData, NeXusName
-from ess.reduce.time_of_flight import GenericTofWorkflow
-
-from .. import parameter_models
-from ..config.workflows import TimeseriesAccumulator
 from ..core.handler import JobBasedPreprocessorFactoryBase
 from ..core.message import StreamId, StreamKind
 from .accumulators import Accumulator, CollectTOA, Cumulative, MonitorEvents
-from .monitor_workflow_specs import MonitorDataParams, MonitorRatemeterParams
-from .stream_processor_workflow import StreamProcessorWorkflow
+from .monitor_workflow_specs import MonitorDataParams
 from .workflow_factory import Workflow
 
-# Type aliases for monitor ratemeter workflow
-CustomMonitor = NewType('CustomMonitor', int)
-CurrentRun = NewType('CurrentRun', int)
-MonitorCountsInInterval = NewType('MonitorCountsInInterval', sc.DataArray)
-
 
 class MonitorStreamProcessor(Workflow):
-    def __init__(self, edges: sc.Variable) -> None:
+    def __init__(
+        self,
+        edges: sc.Variable,
+        *,
+        toa_range_enabled: bool = False,
+        toa_range: tuple[sc.Variable, sc.Variable] | None = None,
```

Comment:
Can you make do with just one argument? If `toa_range` is `None`, it means toa_range is not enabled?